lms/djangoapps/verify_student/services.py

```diff
@@ -9,6 +9,7 @@
 from django.db import IntegrityError
 
 from opaque_keys.edx.keys import CourseKey
+
```

Comment:
I have separated `opaque_keys` import because it's code live in a separate repository (third party app).